tools/pyosmium-get-changes

```diff
@@ -185,7 +205,15 @@ if __name__ == '__main__':
             or 'https://planet.osm.org/replication/minute/'
     logging.info("Using replication server at %s" % url)
 
+    # According to the documentation, the cookie jar loads the file only if FileCookieJar.load is called.
+    cookie_jar = cookiejarlib.MozillaCookieJar(options.cookie)
+    if options.cookie is not None:
+        cookie_jar.load(options.cookie)
+    opener = urlrequest.build_opener(urlrequest.HTTPCookieProcessor(cookie_jar))
```

Comment:
These four lines are only needed when `cookie is not None`, so join with open_url line below.